src/gfn/actions.py

```diff
@@ -425,8 +425,7 @@ def make_dummy_actions(
             A GraphActions object with the specified batch shape filled with dummy
             actions.
         """
-        # TODO: make default dtype int32
-        tensor = torch.zeros(batch_shape + (4,), dtype=torch.int64, device=device)
+        tensor = torch.zeros(batch_shape + (4,), dtype=torch.long, device=device)
```

Comment:
I think the TODO is important; int32 are faster than int64 on GPU

Reply:
What do you think of just putting everything to int32? I actually had it like this at one version of the PR - but I switched it because the default is int64 for pytorch in general so I was worried it was a potential footgun.